src/vscode_ocaml_platform.ml

```diff
@@ -54,6 +54,10 @@ let notify_configuration_changes instance =
     ()
 
 let activate (extension : ExtensionContext.t) =
+  let listeners = Process.listeners "uncoughtException" in
+  let _ =
+    Process.removeListener (Array.get listeners (Array.length listeners - 1))
```

Comment:
My thinking that this code should go right after `include Base` in `import.ml`. The reason is that we'd like to remove the listener as soon as it's added to make sure we aren't removing someone else's listener by accident. Do you agree?

Also, a comment explaining the problem would be quite handy.

Reply:
move to `import.ml` and add comment

Reply:
Awesome. Thanks.

Just to double check, you confirmed that this fixes the issue? I have no way of reproducing this unfortunately.

Reply:
Unfortunately, this does not remove the listener as expected.

<img width="1616" alt="Screenshot 2023-09-21 at 21 30 48" src="https://github.com/ocamllabs/vscode-ocaml-platform/assets/71200607/58523ee7-5ede-4c9f-b93c-dda18d3daf7c">

As you can see from the above debugger output, uncaughtException event listener hasn't been removed, and the `listeners` variable does not contain the expected listener functions.

I have no idea why this happens, maybe it's a bug of jsoo or something?

However, I've figured out another way to do the same thing, and that works. I've pushed the valid way to the branch.